src/compas_rhino/uninstall.py

```diff
@@ -45,13 +45,25 @@ def uninstall(version=None, packages=None):
     if version not in ('5.0', '6.0', '7.0'):
         version = '6.0'
 
-    packages = _filter_installed_packages(version, packages)
-
-    ipylib_path = compas_rhino._get_ironpython_lib_path(version)
     # We install COMPAS packages in the scripts folder
     # instead of directly as IPy module.
     scripts_path = compas_rhino._get_scripts_path(version)
 
+    # This is for old installs
+    ipylib_path = compas_rhino._get_ironpython_lib_path(version)
+
+    # If no specific packages are provided for uninstall
+    # everything should be removed,
+    # and not just the default packages
+    # or the packages that the bootstrapper is aware of.
+    if not packages:
+        packages = []
+        for name in os.listdir(scripts_path):
+            if name.startswith('compas') and not name.endswith('.py'):
```

Comment:
again, `roslibpy` or other default compas-installed packages?

Reply:
is this also the case here?

Reply:
yes, i think so.